src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/Application.vue

```diff
@@ -93,6 +93,9 @@ export default defineComponent({
           case "Education":
           case "WorkReferences":
           case "CharacterReferences":
```

Comment:
Can we combine?

```
case "CharacterReferences":
case "Review":
            this.saveDraftAndAlertSuccess();
            this.incrementWizard();
```

Reply:
Ahh nice catch. I didn't realize we were combining all the cases.